### PR TITLE
Aws-ha test case

### PR DESCRIPTION
**Pull request by Horiodino**
# Tasks description

### Related Issue(s)
- https://github.com/ksctl/ksctl/issues/324

# Note to reviewers

- [x] Ran Tests locally
- [x] Checked [Contribution's guidelines](https://docs.ksctl.com/docs/contribution-guidelines/)

### CODE REVIEW

File: internal/cloudproviders/aws/network.go
Comment (dipankardas011) [inline internal/cloudproviders/aws/network.go:265]: Make sure these assigned empty_str during deletion

**Comment by dipankardas011** [comment]
@Horiodino can you check why is this stuck?

image

File: internal/cloudproviders/aws/aws_test.go
Comment (dipankardas011) [inline internal/cloudproviders/aws/aws_test.go:136]: Not sure why is this removed?

**Review by dipankardas011 — CHANGES_REQUESTED** [review]
Changes needed

**Comment by dipankardas011** [comment]
@Horiodino I have changed the base of the test suite to the pr branch of error handling so that we can merge this to that and make aws changes once this is merged to that PR

File: internal/cloudproviders/aws/aws_test.go
Comment (dipankardas011) [reply internal/cloudproviders/aws/aws_test.go:136]: also check what all where already there and keep those unit test cases

File: internal/cloudproviders/aws/awsgo.go
Comment (Horiodino) [reply internal/cloudproviders/aws/awsgo.go:729]: na because the method itself needs pointer

**Comment by dipankardas011** [comment]
@Horiodino have you rebeased it?

**Pull request merged**
